### PR TITLE
add easy diff for document tracking

### PR DESCRIPTION
**Pull request by zhang2014**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category :
- Documentation

Detailed description (optional):

for example:
```
☁  tools [docs/diff_script] ⚡ ./easy_diff.py zh/data_types
```
use stdout print : 
```
Need translate document:/Users/coswde/Source/ClickHouse/docs/tools/../../docs/en/data_types/uuid.md
```
use git pager print: 
````
diff --git a/docs/en/data_types/domains/ipv6.md b/docs/en/data_types/domains/ipv6.md
index 1bfbe3400b..e2abaff017 100644
--- a/docs/en/data_types/domains/ipv6.md
+++ b/docs/en/data_types/domains/ipv6.md
@@ -4,13 +4,13 @@
 
 ### Basic Usage
 
-``` sql
+```sql
 CREATE TABLE hits (url String, from IPv6) ENGINE = MergeTree() ORDER BY url;
 
 DESCRIBE TABLE hits;
 ```
 
-```
+```text
 ┌─name─┬─type───┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┐
 │ url  │ String │              │                    │         │                  │
 │ from │ IPv6   │              │                    │         │                  │
@@ -19,19 +19,19 @@ DESCRIBE TABLE hits;
 
 OR you can use `IPv6` domain as a key:
 
-``` sql
+```sql
 CREATE TABLE hits (url String, from IPv6) ENGINE = MergeTree() ORDER BY from;
... MORE
````

### CODE REVIEW

**Comment by zhang2014** [comment]
This script is based on the assumption that documents in other languages are fully synchronized with the en document at a commit.

File: docs/tools/easy_diff.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import os, sys
+import subprocess
+from git import cmd
+from tempfile import NamedTemporaryFile
+
+SCRIPT_PATH = os.path.abspath(__file__)
+CLICK_HOUSE_REPO_HOME = os.path.join(os.path.dirname(SCRIPT_PATH), '../../')
+
+
+def diffFile(reference_file, working_file, git, temp_diff):
```
Comment (blinkov) [inline docs/tools/easy_diff.py]:
```suggestion
def diff_file(reference_file, working_file, git, temp_diff):
```
snake_case style is more common in python for function names and as far as I remember nearby scripts use it, let's change this? (here and below)

File: docs/tools/easy_diff.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, sys
+import subprocess
+from git import cmd
+from tempfile import NamedTemporaryFile
+
+SCRIPT_PATH = os.path.abspath(__file__)
+CLICK_HOUSE_REPO_HOME = os.path.join(os.path.dirname(SCRIPT_PATH), '../../')
```
Comment (blinkov) [inline docs/tools/easy_diff.py]: not so important, but I think:
*  it's better to write `CLICKHOUSE` in one word
* `os.path.join(os.path.dirname(SCRIPT_PATH), '..', '..')` looks better

File: docs/tools/easy_diff.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+def diffDirectory(reference_directory, working_directory, git, temp_diff):
+    if not os.path.isdir(reference_directory):
+        raise RuntimeError('The [' + reference_directory + '] is not directory.')
+
+    for list_item in os.listdir(reference_directory):
+        working_item = os.path.join(working_directory, list_item)
+        reference_item = os.path.join(reference_directory, list_item)
+        if diffFile(reference_item, working_item, git, temp_diff) if os.path.isfile(reference_item) else diffDirectory(reference_item, working_item, git, temp_diff) != 0:
+            return 1
```
Comment (blinkov) [inline docs/tools/easy_diff.py]: shouldn't we use this number as exit code below where we call these functions?

File: docs/tools/easy_diff.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    if os.path.samefile(os.path.join(CLICK_HOUSE_REPO_HOME, 'docs'), custom_document):
+        return os.path.join(CLICK_HOUSE_REPO_HOME, 'docs', other_language, *children[1:])
+    children.insert(0, os.path.split(custom_document)[1])
+    return findLanguageDoc(os.path.split(custom_document)[0], other_language, children)
+
+
+if __name__ == '__main__':
+    git = cmd.Git(CLICK_HOUSE_REPO_HOME)
+    git_pager = git.execute(['git', 'var', 'GIT_PAGER'])
+    working_language = os.path.join(CLICK_HOUSE_REPO_HOME, 'docs', sys.argv[1])
```
Comment (blinkov) [inline docs/tools/easy_diff.py]: We'd better start using `argparse` module right away to make it more flexible and extendable. For example there could be an option to skip the git pager part.

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by blinkov** [comment]
> This script is based on the assumption that documents in other languages are fully synchronized with the en document at a commit.

@zhang2014 it's better to add this comment to `--help` output or at least in code comments

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**